### DISCUSSION TITLE
Make DataListItem not sealed

### DISCUSSION THREAD

**Discussion by soreng** [leekelleher/umbraco-contentment#279] [Question] [answered]
Hi,

When reading https://skrift.io/issues/creating-an-author-picker-using-contentment/ it seems a bit off to use the icon-property to store an image. Would it be possible to remove the "sealed" part of the class, letting us expose more data to list items that may be used in templated editors?

**Comment by leekelleher** [comment]
Thanks for suggesting this @soreng. My general position (on Contentment) has been to mark classes as `sealed` by default, then open them up as needed - or when someone asks, and now you have. 😀 

I'm toying with an idea to introduce an `IDataListItem` interface, but if I changed the `IDataListSource.GetItems()` signature, that'd be a breaking-change... so might wait for Contentment v5 for that.  In the meantime, unsealing `DataListItem` should be fine. 👍

**Comment by leekelleher** [comment, marked as answer]
This has been resolved in PR #280.
